scripting/src/test/java/org/infinispan/scripting/ClusteredScriptingTest.java

```diff
@@ -58,6 +52,14 @@ public void testDistExecScript() throws IOException, InterruptedException, Execu
       assertTrue(results.contains(manager(1).getAddress()));
    }
 
+   @Test(expectedExceptions = IllegalStateException.class, expectedExceptionsMessageRegExp = "^ISPN026009.*")
```

Comment:
Could you please update this one as well? Having identifiers here might be fragile.